pyswarms/backend/handlers.py

```diff
@@ -574,11 +574,11 @@ def __init__(self, strategy):
             from pyswarms.backend.handlers import OptionsHandler
 
 
-            oh = OptionsHandler(strategy=(None, None, "exp_decay"))
+            oh = OptionsHandler(strategy={ "w":'exp_decay', "c1":'nonlin_mod',"c2":'lin_variation'})
```

Comment:
@nikhil-sethi nit: just run `black` here to convert single-quotes to double-quotes

Reply:
Sure. Will do it.